docs/development/local-development.md

```diff
@@ -23,15 +23,14 @@ You can use the following commands on Ubuntu.
 
 ```sh
 curl -sL https://deb.nodesource.com/setup_18.x | sudo -E bash -
```

Comment:
Should this point to Node.js 20, maybe?

`https://deb.nodesource.com/setup_18.x`

Reply:
sure, but unrelated to this pr

Reply:
no, we still recommend to use latest lts which is v18 for now